frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1342,28 +1342,31 @@ class Worksheet extends React.Component {
                         // the deleted bundle is the only item of the table
                         if (this.state.subFocusIndex === 0) {
                             // the deleted item is the last item of the worksheet
-                            if (items.length === focus + 1) {
+                            if (blocks.length === focus) {
```

Comment:
This line fixes it, but the logic here originally only accounted for deleting one bundle, the whole part needs a refactor